cli/tests/pcluster/api/controllers/test_cluster_operations_controller.py

```diff
@@ -1041,6 +1063,7 @@ def test_incompatible_version(self, client, mocker):
             "cluster_creation_timeout",
             "cluster_resource_creation_failure",
             "no_stack_event",
+            "cluster_protected_mode"
```

Comment:
missing comma at the end (this might be already caught by the Github checks)?

Reply:
Yes, Tox has 2 warnings on the comma. Fixed.